src/StackAdmin/Azs.Fabric.Admin/Module/Azs.Fabric.Admin/Azs.Fabric.Admin.psd1

```diff
@@ -126,9 +127,52 @@ Licensed under the MIT License. See License.txt in the project root for license
             # IconUri = ''
 
             # ReleaseNotes of this module
```

Comment:
You only need notes for the current release.

Reply:
Fixed.